### ISSUE TITLE
[Design] Create wireframes for VAHB error states

### ISSUE THREAD

**Issue by cgednalske** [department-of-veterans-affairs/va.gov-team#99657]

**Comment by outerpress** [comment]
@cgednalske i'm wondering if this one can be closed out or moved to the icebox? It's pretty old, are we still planning on doing this work someday?

**Comment by cgednalske** [comment]
@outerpress I would keep all the open tickets from the [related epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/94166#issue-2562515914) but move them to the icebox. Would that work? That way if/when this work is picked up again someday (which I think is still the plan for Pre-check-in) the next steps are in place.

**Comment by outerpress** [comment]
Yep that sounds good to me @cgednalske - I'll move that epic to the priorities Icebox on our board too. Thanks!
